### PR TITLE
everything completed

### PR DESCRIPTION
**Pull request by patryKKRK**
everything completed, please review

### CODE REVIEW

**Comment by GrzegorzChwastarz** [comment]
1 You don't need to specify ID of the resources by variables. Terraform (AWS) creates it on our behalf so please remove all the variables and occurrences inside the resources. If you need to reference one resource from another use do it like `aws_subnet. private2.id`

2 Unspoken rule while creating module is that module variables names are the same as the ones inside the resource. So it would be for example 
```
resource "aws_subnet" "private2" {
  vpc_id            = var.vpc_id
  cidr_block        = var. cidr_block[1]
  availability_zone = var.availability_zone

  tags = {
    Name = "private2"
  }
} 
```

That's for starters :)

File: module/output.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
+output "vpc_id" {
+  value = aws_vpc.vpc1.id
+}
+output "public_output_1" {
+  value = aws_subnet.public1.id
+
```
Comment (GrzegorzChwastarz) [inline module/output.tf:6]: Try to be consistent with formatting. Avoid additional lines and make it look the same everywhere :)

File: module/output.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  value = aws_eip.ip2.id
+}
+
+output "ig_id" {
+  value = aws_internet_gateway.gw.id
+}
+
+output "rt1" {
+  value = aws_route_table.internet_gateway_route.id
+
```
Comment (GrzegorzChwastarz) [inline module/output.tf:27]: extra line

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  }
+}
+provider "aws" {
+  profile = "chaos"
+  region  = "eu-central-1"
+}
+
+module "networking" {
+  source   = "./module"
+
```
Comment (GrzegorzChwastarz) [inline main.tf:16]: remove extra line

File: module/variables.tf [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    private_subnet_1 = "private1"
+    private_subnet_2 = "private2"
+    public_subnet_1  = "public1"
+    public_subnet_2  = "public2"
+    vpc              = "vpc1"
+
+  }
+}
+
+variable "tag" {
```
Comment (GrzegorzChwastarz) [inline module/variables.tf:28]: variable is not used anywhere, can be removed.

**Comment by DominikGrzywaczewski** [comment]
Still I don't see configuration for pre-commit hooks based on https://github.com/antonbabenko/pre-commit-terraform (it's really useful solution and helps a lot with daily basis coding fun)

**Pull request merged**